xml/ay_bootloader.xml

```diff
@@ -260,6 +220,61 @@
      </para>
     </listitem>
    </varlistentry>
+   <varlistentry>
+    <term>generic_mbr</term>
+    <listitem>
+     <para>
+      Write generic boot code to the MBR (will be ignored if
+      <literal>boot_mbr</literal> is set to <literal>true</literal>).
+     </para>
+<screen>&lt;generic_mbr config:type="boolean"&gt;false&lt;/generic_mbr&gt;</screen>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>gfxmode</term>
+    <listitem>
+     <para>
+      Graphical resolution of the &grub; screen (requires &lt;terminal&gt; to
+      be set to <literal>gfxterm</literal>).
+     </para>
+     <para>
+      Valid entries are <literal>auto</literal>, <literal><replaceable>
+      HORIZONTAL</replaceable>x<replaceable>VERTICAL</replaceable></literal>,
+      or <literal><replaceable>HORIZONTAL</replaceable>x<replaceable>VERTICAL
+      </replaceable>x<replaceable>COLOR DEPTH</replaceable></literal>. You can
+      see the screen resolutions supported by &grub; on a particular system by
+      using the <command>vbeinfo</command> command at the &grub; command line
+      in the running system.
+     </para>
+<screen>&lt;gfxmode&gt;1280x1024x24&lt;/gfxmode&gt;</screen>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>os_prober</term>
+    <listitem>
+     <para>
+      If set to <literal>true</literal>, automatically searches for operating
+      systems already installed and generates boot entries for them during the
+      installation.
+     </para>
+<screen>&lt;os_prober&gt;false&lt;/os_prober&gt;</screen>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>password</term>
+    <listitem>
+     <para>
+      If defined, then it protects boot via password. It has three subelements.
```

Comment:
type "via" should be "with" probably.

Reply:
Does this protect the boot menu? If it does, it should say something like "If defined, it protects the boot menu with a password. It has three sub-elements:"

Reply:
It is more tricky. See https://documentation.suse.com/sles/15-SP3/single-html/SLES-administration/#sec-grub2-yast2-config-advanced-boot-loader
It really depends if you use unrestricted or not. If you do not use it, then it will block boot. If you use it, then it just protects about modifying boot.

Reply:
ok, then the reference to grub2.xml  will take care of this.